cc_corpus/corpus.py

```diff
@@ -363,3 +407,31 @@ def close(self):
     def __del__(self):
         """Just calls close()."""
         self.close()
+
+
+def convert_document_to_json(document: Document):
```

Comment:
I think I have found the final place for this function. Converting a `Document` to "corpus format" is done via its `__str__` method. Let's add the code below to a new `to_json` method.

Reply:
Ok.